code/game/objects/structures/grey_autocloner.dm

```diff
@@ -0,0 +1,145 @@
+/obj/machinery/grey_autocloner
+	name = "technocracy cloning pod"
+	desc = "An advanced Technocracy cloning pod, used for rapid cloning. Very delicate, very power hungry, and very much should not be tampered with."
+	icon = 'icons/obj/abductor.dmi'
+	icon_state = "experiment-open"
+	anchored = TRUE
+	density = TRUE
+	var/mob/living/carbon/human/occupant
+	var/datum/gas_mixture/air_contents = new()
+	var/list/brine_types = list("corazone", "perfluorodecalin", "epinephrine", "salglu_solution") //Taken from cloner, a bit of extra healing though they should be fully good.
+	var/datum/mind/clonemind
+	var/attempting = FALSE
+
+/obj/machinery/grey_autocloner/attackby(obj/item/implanter/implant, mob/user, params)
+	if(!istype(implant) || !(istype(implant.imp, /obj/item/implant/grey_autocloner)))
+		return ..()
+	var/obj/item/implant/grey_autocloner/autoclone= implant.imp
+	autoclone.linked = src
+	atom_say("Link confirmed!")
+
+/obj/machinery/grey_autocloner/Initialize(mapload)
+	. = ..()
+
+	air_contents.oxygen = MOLES_O2STANDARD * 2
+	air_contents.nitrogen = MOLES_N2STANDARD
+	air_contents.temperature = T20C
+
+/obj/machinery/grey_autocloner/Destroy() //These will only be not null if deleted with someone in it
+	occupant = null
+	clonemind = null
+	return ..()
```

Comment:
move these to right after the type definition

Reply:
done